### DISCUSSION TITLE
Module not found Error: can't resolve 'next/image' in '/User/***/***'

### DISCUSSION THREAD

**Discussion by hpachy** [vercel/next.js#22698] [Help]
### What example does this report relate to?

next/image

### What version of Next.js are you using?

10.0.7

### What version of Node.js are you using?

15.1.0

### What browser are you using?

chrome

### What operating system are you using?

macOS

### How are you deploying your application?

next build

### Describe the Bug

Hello I got the same problem since 1 day... I'm sharing with you my json package to figured out what's going on

```json
{
  "name": "showcase-front",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start"
  },
  "dependencies": {
    "next": "10.0.7",
    "react": "17.0.1",
    "react-device-detect": "^1.15.0",
    "react-dom": "17.0.1",
    "react-modal": "^3.12.1",
    "react-use": "^17.1.1"
  },
  "devDependencies": {
    "@types/node": "^14.14.31",
    "@types/react": "^17.0.2",
    "typescript": "^4.1.5"
  }
}
```
$ next build
>Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db
info  - Creating an optimized production build  
Failed to compile.

ModuleNotFoundError: Module not found: Error: Can't resolve 'next/image' in '/Users/me/***/***/components/HomePage'

> Build error occurred
Error: > Build failed because of webpack errors
    at build (/usr/local/lib/node_modules/next/dist/build/index.js:15:918)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:93:5)

### Expected Behavior

I expected to build correctly with out any error of next/image 

### To Reproduce

I changed SVG case...

**Comment by jamesmosier** [comment]
Can you share a full reproduction of this issue? i.e. a GitHub repository
